### DISCUSSION TITLE
Task history not updating for newly scheduled payments for new wallets

### DISCUSSION THREAD

**Discussion by polkadirk** [OAK-Foundation/OAK-blockchain#493] [Automation Hub]
I created a new wallet with pjs and transferred 200 TUR then created three recurring payments to another wallet. The transfers executed successfully but the task history never showed any active tasks and still does not show completed tasks. The wallet id is `6Ait2WaNJ5wVPtSN88pvWnTj9JxYwVfm9ypNaFPKD1riHwTc` and the corresponding successfull subscan transfers can be found here - https://turing.subscan.io/account/6Ait2WaNJ5wVPtSN88pvWnTj9JxYwVfm9ypNaFPKD1riHwTc?tab=transfer

I created a second new wallet and recreated the issue on video and again there was no update to the task history or task overview tabs.

https://github.com/OAK-Foundation/OAK-blockchain/assets/104081857/4704876f-a863-44cd-8f2d-077bcdc294e3

**Comment by v9n** [comment]
this should be fixed now @polkadirk
